src/utils/useScrollListener.ts

```diff
@@ -2,34 +2,24 @@ import { useEffect, useRef, useState } from 'react';
 
 export function useScrollThresholdListener(threshold: number, debounce = 300) {
   const [isAboveThreshold, setIsAbove] = useState(false);
-  const [isDebouncing, setIsDebouncing] = useState(false);
```

Comment:
Not sure entirely about removing the debouncing variable, by doing so it causes certain threshold to just call handleScroll in a loop

Reply:
Yeah, as @Xaroz said, this debounce checking was to avoid an infinite loop where the header animation would change size, triggering the listener, which would change the size, etc.

Thanks @LyDe17TC for the contribution but we'll leave this hook as it is.

Reply:
no problem, if I can help you don't hesitate